reviews/methods1.Rmd

```diff
@@ -0,0 +1,450 @@
+The authors seem to argue that even in exploratory studies preregistration will remove *some* uncertainty about the ability of a test to falsify a prediction.
+That is true.
+However, they do not argue why it would do so to an extent that makes preregistration useful in real life. If an action I do reduces my uncertainty from 98% to 94%, it has relatively reduced my uncertainty.
+At the same time, it has done so in a manner that does not matter at all.
+My point here is that the error rate in exploratory research is inflated above alpha% to an unknown extent.
+A preregistration might reduce the inflation somewhat.
+But the rate is quickly too high so that I wouldn't trust the claim anyway.
+
+> Actually, we think this is one of the strengths of the paper.
+> With the mathematical analysis, we show exactly how much uncertainty, theoretical risk, and detectability result in how much confirmation, as seen in Figures 1 and 2.
+> The vague intuition of a "[Type 1 error] rate [that] is quickly too high so that I wouldn't trust the claim anyway" is replaced by a clear functional form that enables navigating tradeoffs involving exploration, post-hoc changes or detectability.
+
+If a claim is made with 10% lower Type 1 error rate than it would have been made when it was not preregistered is not useful - because I do not know what the error rate associated with the claim is.
+
+> Our claim is that uncertainty is reduced, not theoretical risk directly.
+> We claim that there are exploratory methods with a well-known error rate.
+> For example, a neural network has extreme flexibility in fitting the data closely, yet it is calculatable.
+
+Thus, preregistration in exploratory studies reduces the error rate inflation, but it does not make the error rate known.
+Therefore, I do not believe preregistration in exploratory studies is useful, and the authors do not make an argument for it.
+We do not want to reduce the error rate inflation in preregistration - we want to control it.
+
+> Somehow, our argument seems to be flipped here.
+> An exploratory study has low theoretical risk.
+> A preregistered study has low uncertainty about theoretical risk.
+> A preregistered exploratory study is more persuasive than a study with the same exploration going on that is not preregistered.
+
+If the error rate is not controlled, I do not know how severely a claim has been tested, and I can not use the claim to decide how to act as a scientist.
+The authors seem happy with the fact that preregistration reduces the Type 1 error rate even in an exploratory study.
+
+> No, we are happy with low uncertainty about theoretical risk.
+> An exploratory study is in our framework, signified by low theoretical risk (high Type 1 error rate).
+
+But I am not. Only claims with a known error rate are useful in science.
+
+> Exactly. The error rate must be known (readers must be certain about them).
+> It may be high however (=exploratory).
```

Comment:
Is it possible to construe an example, where there a two studies. One has low estimated error but high uncertainty, the other one has higher error but very low uncertainty; then on average (over many studies in a cumulative scientific process), it would be more effective to run the study with the higher error? Could one directly put this into a Bayesian reasoning showing that the posterior over hundred(or whatever number) such studies would be narrower in the second case?